pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -190,7 +194,7 @@ func (pc *PodController) handleAddUpdatePod(obj interface{}) error {
 	// Avoid processing Pod annotation, if we already have at least one secondary network successfully configured on this Pod.
 	// We do not support/handle Annotation updates yet.
 	if len(podCNIInfo.NetworkConfig) > 0 {
-		klog.InfoS("Secondary network already configured on this Pod. Annotation update not supported.", klog.KObj(pod))
+		klog.InfoS("Secondary network already configured on this Pod. Annotation update not supported.", "Pod", klog.KObj(pod))
```

Comment:
```suggestion
		klog.InfoS("Secondary network already configured on this Pod and annotation update not supported, skipping update", "pod", klog.KObj(pod))
```

(We don't include a period at the end of log messages)

Reply:
yes. took care.